io/driver/module_unload_load.sh

```diff
@@ -16,7 +16,11 @@
 
 CONFIG_FILE="$AVOCADO_TEST_DATADIR"/config
 BUILT_IN_DRIVERS=`cat /lib/modules/$(uname -r)/modules.builtin |awk -F"/" '{print $NF}'|sed 's/\.ko//g'`
-DRIVERS=`find /lib/modules/$(uname -r)/ -name \*.ko | awk -F"/" '{print $NF}'|sed 's/\.ko//g'`
+if [[ $ONLY_IO == True ]]; then
```

Comment:
Yes, when we pass `ONLY_IO : True` from yaml file,
and try to do `if $ONLY_IO; then echo "Test"; fi`, it does not work as expected.
But when we do `if [[ $ONLY_IO == True ]]; then echo "Test"; fi`, works as expected